modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/partstate/GroupPartitionId.java

```diff
@@ -53,7 +53,7 @@ public GroupPartitionId(final int grpId, final int partId) {
      * @return flag to be used for partition
      */
     public static byte getFlagByPartId(final int partId) {
-        return partId == PageIdAllocator.INDEX_PARTITION ? PageMemory.FLAG_IDX : PageMemory.FLAG_DATA;
+        return partId == PageIdAllocator.INDEX_PARTITION ? PageStore.TYPE_IDX : PageStore.TYPE_DATA;
```

Comment:
Why do we return page store type from the method which should return flag?

Reply:
Additional static method for page store type created